pkg/application/create.go

```diff
@@ -384,7 +384,7 @@ func (svc *Service) ValidateCreate(op CreateOp) error {
 			return fmt.Errorf("%s/%s repository already exists", op.OrgName, op.Name)
 		}
 		if response.StatusCode != http.StatusNotFound {
-			return fmt.Errorf("error while checking if %s/%s repository exists", op.OrgName, op.Name)
+			return fmt.Errorf("error while checking if %s/%s repository exists: status code %d, error: %v", op.OrgName, op.Name, response.StatusCode, err)
 		}
```

Comment:
hey, is there a link to existing repo in the err?

Reply:
added github link on error output